src/Storages/IStorage.h

```diff
@@ -173,8 +175,10 @@ class IStorage : public std::enable_shared_from_this<IStorage>, public TypePromo
     virtual NamesAndTypesList getVirtuals() const;
 
     Names getAllRegisteredNames() const override;
-protected:
 
+    NameDependencies getColumnNamesAndReferencedMvMap(const Context & context) const;
```

Comment:
Probably a bad name. I think, something like `getDependentViewsByColumn` will be better.